docs/content/docs/data-structures/counter.md

```diff
@@ -1,5 +1,141 @@
 ---
 title: SharedCounter
-menuPosition: 9
+menuPosition: 5
 status: unwritten
 ---
+
+## Introduction
+
+The `SharedCounter` distributed data structure (DDS) is use to store a integer counter value that can be modified collaboratively.
+The data structure affords incrementing and decrementing the shared value via its [increment](method) (decrements are handled by providing a negative delta value).
+
+The `SharedCounter` is a specialized, [Optimistic DDS]({{< relref "dds.md#optimistic-data-structures" >}}).
+It operates on communicated deltas, rather than direct changes to the shared value.
+In this way, it avoids the pitfalls of DDSes with simpler merge strategies, in which one user's edit may clobber another's.
```

Comment:
This seems like a major point that the article should return to at the end when the `updateCounterValueLabel` function is discussed.

Reply:
I've added a `Why a specialized DDS` section to the counter article to explain this. Definitely let me know what you think. I didn't include any code in this example, but I can definitely add something if you think it would add value.